fiat-roles/src/main/java/com/netflix/spinnaker/fiat/providers/DefaultAccountProvider.java

```diff
@@ -36,18 +36,19 @@
 
   @Autowired
   public DefaultAccountProvider(ClouddriverService clouddriverService) {
+    super();
     this.clouddriverService = clouddriverService;
   }
 
   @Override
-  public Set<Account> getAll() throws ProviderException {
+  protected Set<Account> loadAll() throws ProviderException {
     try {
       val returnVal = clouddriverService.getAccounts().stream().collect(Collectors.toSet());
       success();
       return returnVal;
-    } catch (RetrofitError re) {
+    } catch (Exception e) {
```

Comment:
Any exception should register as a failure - it's then converted to a `ProviderException` in BaseProvider